<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadUI</class>
 <widget class="QWidget" name="DownloadUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widgetImage" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="labelImage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true">Image</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QWidget" name="widgetChangeImageButton" native="true">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayoutButtonSlider"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widgetDownload" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labelProgressBarFile">
            <property name="styleSheet">
             <string notr="true">color : @color-text-gray;</string>
            </property>
            <property name="text">
             <string notr="true">labelDownloadFile</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarFile">
            <property name="styleSheet">
             <string notr="true">QProgressBar:horizontal { 
	 background: #364058; 
	 border-radius: 3px;
	padding: 3px; 
	 margin: 5px 10px 5px 10px;
	 height: 2px;
}
QProgressBar::chunk:horizontal {
	 background: #7dd1ee;
}</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="textDirection">
             <enum>QProgressBar::TopToBottom</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>20</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="labelProgresseBarGlobal">
                 <property name="styleSheet">
                  <string notr="true">color : @color-text-gray;</string>
                 </property>
                 <property name="text">
                  <string notr="true">labelDownloadGlobal</string>
                 </property>
                </widget>
               </item>
               <item alignment="Qt::AlignRight">
                <widget class="QLabel" name="label">
                 <property name="minimumSize">
                  <size>
                   <width>250</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color : @color-text-gray;</string>
                 </property>
                 <property name="text">
                  <string notr="true">labelRemainingTime</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBarGlobal">
            <property name="styleSheet">
             <string notr="true">QProgressBar:horizontal { 
	 background: #364058; 
	 border-radius: 3px;
	padding: 3px; 
	 margin: 5px 10px 5px 10px;
	 height: 2px;
}
QProgressBar::chunk:horizontal {
	 background: #7dd1ee;
}</string>
            </property>
            <property name="value">
             <number>24</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
